packages/gatsby-theme-newrelic/README.md

```diff
@@ -9,91 +9,92 @@ websites](https://opensource.newrelic.com).
+      - [`robots`](#robots)
+      - [`relatedResources`](#relatedresources)
+      - [`tessen`](#tessen)
+      - [`resolveEnv`](#resolveenv)
+      - [`i18n`](#i18n)
+      - [`layout`](#layout)
+      - [`prism`](#prism)
+      - [`gaTrackingId`](#gatrackingid)
+      - [`splitio`](#splitio)
+        - [Environment-specific configuration](#environment-specific-configuration)
+  - [Components](#components)
+    - [`Banner`](#banner)
+    - [`Button`](#button)
+    - [`Callout`](#callout)
+    - [`CodeBlock`](#codeblock)
+    - [`Collapser`](#collapser)
+    - [`CollapserGroup`](#collapsergroup)
+    - [`ContributingGuidelines`](#contributingguidelines)
+    - [`CookieConsentDialog`](#cookieconsentdialog)
+    - [`Dropdown`](#dropdown)
+      - [`Dropdown.Toggle`](#dropdowntoggle)
+      - [`Dropdown.Menu`](#dropdownmenu)
+      - [`Dropdown.MenuItem`](#dropdownmenuitem)
+    - [`ExternalLink`](#externallink)
+    - [`FeatherSVG`](#feathersvg)
+    - [`Feedback`](#feedback)
+    - [`GlobalFooter`](#globalfooter)
+    - [`GlobalHeader`](#globalheader)
+    - [`HamburgerMenu`](#hamburgermenu)
+    - [`Icon`](#icon)
+      - [Available icons](#available-icons)
+      - [Shadowing icons](#shadowing-icons)
+    - [`Layout`](#layout-1)
+      - [`Layout.Content`](#layoutcontent)
+      - [`Layout.Footer`](#layoutfooter)
+      - [`Layout.Main`](#layoutmain)
+      - [`Layout.PageTools`](#layoutpagetools)
+      - [`Layout.Sidebar`](#layoutsidebar)
+    - [`Link`](#link)
+    - [`MarkdownContainer`](#markdowncontainer)
+    - [`MDX`](#mdx)
+      - [Using `MDX`](#using-mdx)
+      - [Default components](#default-components)
+    - [`MDXCodeBlock`](#mdxcodeblock)
+    - [`NavItem`](#navitem)
+      - [`Page`](#page)
+    - [`NewRelicLogo`](#newreliclogo)
+    - [`PageTools`](#pagetools)
+      - [`PageTools.Section`](#pagetoolssection)
+      - [`PageTools.Title`](#pagetoolstitle)
+    - [`RelatedResources`](#relatedresources-1)
+    - [`SearchInput`](#searchinput)
+    - [`SEO`](#seo)
+    - [`SimpleFeedback`](#simplefeedback)
+    - [`Spinner`](#spinner)
+    - [`Surface`](#surface)
+    - [`Table`](#table)
+    - [`Tag`](#tag)
+    - [`TagList`](#taglist)
+    - [`Terminal`](#terminal)
+    - [`Video`](#video)
+  - [MDX Component variants](#mdx-component-variants)
+  - [Hooks](#hooks)
+    - [`useClipboard`](#useclipboard)
+    - [`useFormattedCode`](#useformattedcode)
+    - [`useKeyPress`](#usekeypress)
+    - [`useLayout`](#uselayout)
+    - [`useQueryParams`](#usequeryparams)
+    - [`useTessen`](#usetessen)
+    - [`useTimeout`](#usetimeout)
+    - [`useUserId`](#useuserid)
+    - [`usePrevious`](#useprevious)
+  - [I18n](#i18n-1)
+  - [Announcements](#announcements)
+  - [Utils](#utils)
+    - [`formatCode`](#formatcode)
+  - [Testing](#testing)
+    - [Changes to Jest config](#changes-to-jest-config)
+    - [Mocking `ReactDOM.createPortal` for snapshot testing](#mocking-reactdomcreateportal-for-snapshot-testing)
```

Comment:
Do you know why there are so many changes to this TOC?

Reply:
@jerelmiller i have a TOC tool I use to auto gen TOCs from markdown https://github.com/jonschlinkert/markdown-toc so it's hooked into VSCode..